lib/rentlinx/client.rb

```diff
@@ -0,0 +1,32 @@
+require 'httpclient'
+require 'json'
+require 'uri'
+require 'rentlinx/default'
+
+module Rentlinx
+  # Client is the primary interface that users of this gem will utilize.
+  class Client
+    def initialize(username, password, api_url_prefix)
+      @url_prefix = (api_url_prefix + '/').freeze  # Extra slashes are fine
```

Comment:
I would consider extra slashes to be an error. Most of the time it's fine but there are a few cases where it can trip up certain webservers. If we're hoping others will implement a Rentlinx capable API we should go perfection :smile:

Plus it could be fixed by replacing `+ '/'` with `.gsub(/\w$/, '\0/')` so it's probably good to just do that.